### PR TITLE
Lesson 14

### PR DESCRIPTION
**Pull request by dero5001**
I create 2 programs for tusk 3. 
3.1 - easyer if we have usual masterdata.
3.2 - if we have a text with some e-mails that we need to encode.

### CODE REVIEW

File: Lesson_14/Tusk_1.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        else:
+            print(len(phone_list))
+
+    elif command == 'add':
+        name = input('Please enter a name: ')
+        if name in phone_list:
+            print('This name is already here')
+            continue
+
+        while True:
```
Comment (klymenok) [inline Lesson_14/Tusk_1.py]: I don't think `while True` is a good choice here. Maybe it is better to skip this command and start from the beginning when a user enters the wrong format?

File: Lesson_14/Tusk_1.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+            phone_nr = input('Please enter a number: ')
+
+            if re.match('\+380[\d]{9}$', phone_nr):
```
Comment (klymenok) [inline Lesson_14/Tusk_1.py]: Also, I would suggest making it simpler and don't overcomplicate:
1. Do not rewrite the original phone number. If user entered `09812345678` let it be the same format 
2. Try to implement DRY principle (Don't Repeat Yourself): here you have 3 conditions with the same code everywhere. So basically, only conditions are different. You can use it and save some space:
```
if re.match(...) or re.match(...) or re.match(...):
    phone_list[name] = phone_nr
    json_data = json.dumps(phone_list)

     with open(file_address, 'w') as file:
         file.write(json_data)
     print(f'The name {name} added')

```

Comment (dero5001) [reply Lesson_14/Tusk_1.py]: yes, it looks better, thank you.

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**